protocols/syslog/syslog.c

```diff
@@ -34,88 +35,142 @@
 #include "syslog_net.h"
 
 
-static char send_buffer[MAX_DYNAMIC_SYSLOG_BUFFER + 1];
+static char send_buffer[MAX_LINES_SYSLOG_BUFFER][MAX_LENGTH_SYSLOG_BUFFER + 1];
 extern uip_udp_conn_t *syslog_conn;
 static struct SyslogCallbackCtx syslog_callbacks[SYSLOG_CALLBACKS];
+int8_t act_char_line = -1;
 
-static void syslog_send_cb_P(void *data) 
+static void
+syslog_send_cb_P(void *data)
 {
   strcpy_P(uip_appdata, data);
```

Comment:
What happens if the UIP buffer is smaller than the string? Should we better user strncpy?